Purchases/Public/RCPurchases.m

```diff
@@ -30,7 +30,6 @@
 #import "RCIntroEligibility+Protected.h"
 #import "RCReceiptRefreshPolicy.h"
 #import "RCPurchaserInfoManager.h"
-#import "RCLoggerBridge.h"
```

Comment:
this actually makes me realize that `APITester` isn't currently covering our entire public API - there are public declarations that live in headers other than `RCPurchases.h`. The enum used here was one of them

Reply:
so essentially everything that's [listed here](https://github.com/RevenueCat/purchases-ios/blob/955da7d37c1dff764b2dc4c490dba559bec0b6fb/Purchases/Public/Purchases.h#L20) is a part of our public API and should be tested by API Tester

Reply:
You can see how I'm doing it here: 
#594 
https://github.com/RevenueCat/purchases-ios/pull/594/files#diff-710d630cc91e5f77506dbed62cd6f00991f0d96d367bc437aa3062aa9b307ddb